packages/core-components/src/components/ContentPreview/index.vue

```diff
@@ -44,7 +45,7 @@ export default {
       if (!allowedTypes.length) return containers;
       return containers.map(container => ({
         ...container,
-        elements: container.elements.filter(it => allowedTypes.includes(it.type))
+        elements: intersectionBy(container.elements, allowedTypes, 'type')
```

Comment:
How did this one ended up here 🤔

Reply:
Renato told me that you fixed this bug on VPL but was not propagated on base tailor 😅